pkg/testcommon/testcommon.go

```diff
@@ -241,3 +241,12 @@ func ContainerCheck(checkName string, checkState string) (bool, error) {
 
 	return false, errors.New("unable to find container " + checkName)
 }
+
+// TimeTrack determines the amount of time a function takes to return. Timing starts when it is called.
+// It returns an anonymous function that, when called, will print the elapsed run time.
+func TimeTrack(start time.Time, name string) func() {
+	return func() {
+		elapsed := time.Since(start)
+		log.Printf("%s took %s", name, elapsed)
```

Comment:
Would like to see this have a harvesting key in it, so just add "PERF:" on the front please. Then we *should* be able to harvest it out of the test output with a grep and package it up with the artifacts, into a CSV or something. Alternate ideas are welcome!